### DISCUSSION TITLE
'Points' endpoint returning 'Unexpected Problem'

### DISCUSSION THREAD

**Discussion by rduff** [weather-gov/api#212] [Q&A]
**Describe the bug**
Calls to the "points" API are returning "unexpected problem." The example works, but I have not found another set of coordinates that is functional.

**To Reproduce**
This, for example, is my location:
https://api.weather.gov/points/36.1067,-86.7745

If possible, also include the correlationId from the response.
284805e

**Expected behavior**
A json response containing the forecast endpoints

**Screenshots**
None

**Environment**
Python "requests," Chrome/Version 89.0/MacOS
If you can provide some sample code, that is also helpful.

**Additional context**
None

**Comment by JJKraw** [comment]
> Calls to the "points" API are returning "unexpected problem." The example works, but I have not found another set of coordinates that is functional.

Can you provide examples of where this fails? I use this frequently as the first step of a "find the forecast for this place" process and can't think of a case where it has failed outside of using coordinates outside of the NWS's responsibility.
